apps/src/lib/kits/maker/ui/DiscountCodeInstructions.jsx

```diff
@@ -0,0 +1,71 @@
+import React, {Component, PropTypes} from 'react';
+import i18n from "@cdo/locale";
+import View_Cart from "../../../../../../dashboard/app/assets/images/maker/View_Cart.png";
+import Add_to_cart from "../../../../../../dashboard/app/assets/images/maker/Add_to_cart.png";
+import Enter_Discount_Code from "../../../../../../dashboard/app/assets/images/maker/Enter_Discount_Code.png";
+
+const styles = {
+  title: {
+    fontSize: 32,
+  },
+  image: {
+    width: 400,
+  }
+};
+
+export default class DiscountCodeInstructions extends Component {
+  static propTypes = {
+    discountCode: PropTypes.string.isRequired,
+    discountComplete: PropTypes.bool.isRequired,
```

Comment:
What about something like `fullDiscount`? `discountComplete` could be understood to mean we've completed the discount process rather than we get the whole discount.